facebook_events_importer.js

```diff
@@ -0,0 +1,74 @@
+// ==UserScript==
+// @name        MusicBrainz: Import events from Facebook
+// @version     2016-01-17
+// @author      Ohm Patel
+// @require     https://code.jquery.com/jquery-2.2.0.min.js
```

Comment:
Please add headers as in https://github.com/murdos/musicbrainz-userscripts/blob/master/bandcamp_importer.user.js#L1-L15

Check version format: YYYY.MM.DD.N
Add `@downloadURL`, `@updateURL` and `@icon` headers
Those URLs and correct version are required to allow automatic upgrades.

Adapt `@author`.

Add `// @namespace      https://github.com/murdos/musicbrainz-userscripts`

And a proper description (`@description`)